packages/visualizations/src/components/Table/Cell/Format/GeoFormat.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+    import { tick } from 'svelte';
+    import { WebGlMap } from 'components/Map';
+    import tippy from 'components/utils/tippy';
+
+    import type { WebGlMapOptions, WebGlMapData } from 'components/Map';
+
+    export let rawValue: unknown;
+    export let display = (v: unknown) => v;
+    export let mapOptions: WebGlMapOptions;
+    export let sources: (v: unknown) => WebGlMapData['sources'] = () => ({});
+    export let layers: (v: unknown) => WebGlMapData['layers'] = () => [];
+
+    let tooltipContent;
+    let showMap;
+
+    $: data = {
+        sources: sources(rawValue),
+        layers: layers(rawValue),
+    };
+</script>
+
+<div
+    role="button"
+    tabindex="0"
+    use:tippy={{
+        content: tooltipContent,
+        theme: 'ods-visualization-table',
+        delay: [500, 0],
+        duration: [275, 0],
+        maxWidth: 'none',
+        onTrigger: (instance) => {
+            showMap = true;
+        },
+        onUntrigger: (instance) => {
+            showMap = false;
+        },
+    }}
+>
+    <!-- svelte-ignore a11y-click-events-have-key-events -->
+    <div class="label">{display(rawValue)}</div>
+    <!-- To run a WebGl instance only when tooltip is visible -->
+    {#if showMap}
+        <div bind:this={tooltipContent} class="table-cell-map-container">
+            <WebGlMap options={mapOptions} {data} />
+        </div>
+    {/if}
+</div>
+
+<style lang="scss">
+    .label {
+        cursor: pointer;
+        text-decoration: underline;
+    }
+    :global(.table-cell-map-container) {
+        width: 360px;
+        height: 240px;
```

Comment:
A fixed value seems not like a long term solution. How we gonna deal with responsiveness here ? Is it left for a future PR ?

Reply:
Yes, for the moment we go with that.
We didn't get any design about responsiveness